source/connect.txt

```diff
@@ -49,68 +49,69 @@ Connect
 
 When you open |compass-short|, an initial connection dialog appears:
 
-.. figure:: /images/compass/paste-connection-string.png
+.. figure:: /images/compass/new-connection.png
    :figwidth: 690px
-   :alt: Connection dialog for pasting your connection string
+   :alt: New connection dialog for pasting your connection string
 
-|compass-short| provides two methods to connect to your deployment,
-either providing your deployment connection string or filling in your
-deployment information in specific fields.
+|compass-short| provides multiple methods to connect to your deployment,
+either providing your deployment connection string or through Advanced 
+Connection Options. 
 
-.. tabs::
+Paste Connection String
+~~~~~~~~~~~~~~~~~~~~~~~
 
-   .. tab:: Paste Connection String
-      :tabid: paste-connection-string
+When you provide a connection string, |compass-short| supports
+most :manual:`Connection String Options
+</reference/connection-string/#connection-string-options>`
+supported by MongoDB.
 
-      When you provide a connection string, |compass-short| supports
-      most :manual:`Connection String Options
-      </reference/connection-string/#connection-string-options>`
-      supported by MongoDB.
+By default, |compass-short|'s default 
+:manual:`socketTimeoutMS </reference/connection-string/#urioption.socketTimeoutMS>`
+value is 60000, or 60 seconds. If you are frequently experiencing
+timeouts in |compass-short|, consider setting this option to a
+higher value in your connection string.
 
-      By default, |compass-short|'s default 
-      :manual:`socketTimeoutMS </reference/connection-string/#urioption.socketTimeoutMS>`
-      value is 60000, or 60 seconds. If you are frequently experiencing
-      timeouts in |compass-short|, consider setting this option to a
-      higher value in your connection string.
+For a complete list of the connection string options which
+|compass-short| supports, see the
+`Compass Connection README
+<https://github.com/mongodb-js/connection-model/blob/master/README.md#properties>`__
+on GitHub.
 
-      For a complete list of the connection string options which
-      |compass-short| supports, see the
-      `Compass Connection README
-      <https://github.com/mongodb-js/connection-model/blob/master/README.md#properties>`__
-      on GitHub.
+.. include:: /includes/steps/starting-compass-paste-string.rst
 
-      .. include:: /includes/steps/starting-compass-paste-string.rst
+Advanced Connection Options
+---------------------------
 
-   .. tab:: Fill in Connection Fields Individually
-      :tabid: individual-fields
+The Advanced Connection Options provide additonal forms of connecting
+|compass-short| deployments.  
 
-      .. include:: /includes/steps/starting-compass-individual-fields.rst
+:doc:`/connect/general-connection`
+  Learn how to select a schema for your deployment. 
```

Comment:
I don't know if "select a schema" is the best choice of language here. "Schema" typically refers to the way you model your data. That's how it's described elsewhere in the docs: https://docs.mongodb.com/manual/data-modeling/

However, I see that "schema" is the language used in the Compass UI. I think we may need to run this by the Compass dev team and ask why "schema" was the word chosen for this option, because I don't think it's the clearest word to use. Something like "Connection String Format" would be more appropriate imo.

For now, I suggest we change this line to something like "Learn how to change your connection string format." I'll start a thread in the #compass-dev Slack channel inquiring about the text used in the Compass UI so we can see if it can / will be updated.

Reply:
I updated this to "Connection String Scheme" as was seen in the PR for the change in the #compass-dev Slack.